src/platform.ts

```diff
@@ -27,11 +25,21 @@ function isAlpineLinux(): boolean {
   return !!content && (content.match(/^ID=([^\u001b\r\n]*)/m) || [])[1] === 'alpine';
 }
 
-export function getPlatformInfo(): PlatformInfo {
-  const os = isAlpineLinux() ? 'alpine' : process.platform;
+function getSupportedOS(): SupportedOS {
+  const mapping: { [nodePlatform: string]: SupportedOS } = {
+    linux: isAlpineLinux() ? 'alpine' : 'linux',
+    // TODO: Check whether these aliases are necessary
```

Comment:
for the POC I'm sure they aren't but we can create a ticket to ask this at the right time

Reply:
agreed